wicketd/tests/integration_tests/updates.rs

```diff
@@ -70,8 +70,10 @@ async fn test_updates() {
         .into_inner();
 
     // We should have an artifact for every known artifact kind...
```

Comment:
Maybe add `(except Zone)` like the other comments? I'm not sure how valuable that really is, but it does seem like the comment as written is now incorrect.